docs/environment-variables.md

```diff
@@ -148,6 +150,14 @@ HTTP servers with /routing/v1 endpoints to use for delegated routing (comma-sepa
 
 Default: `https://cid.contact`
 
+### `RAINBOW_DNSLINK_RESOLVERS`
+
+DNS-over-HTTPS servers to use for resolving DNSLink on specified TLDs (comma-separated map: `TLD:URL,TLD2:URL2`).
+
+It is possible to override OS resolver by passing root:  `. : catch-URL`.
+
+Default: `eth. : https://resolver.cloudflare-eth.com/dns-query, crypto. : https://resolver.cloudflare-eth.com/dns-query`
```

Comment:
Keeping the same value as implicit default from boxo for now – just documenting what these are and making them explicit in rainbow's default config.